src/components/Card/index.tsx

```diff
@@ -41,25 +40,35 @@ const BaseCard = ({
 	const Rows =
 		rows &&
 		map(rows, (row, index: number) => (
-			<Box key={index} fontSize={2}>
+			<Flex
+				key={index}
+				fontSize={2}
+				flexDirection="column"
+				flexWrap="wrap"
+				width="100%"
+			>
 				{index > 0 && <Divider />}
 				{row}
-			</Box>
+			</Flex>
 		));
 
 	const Body = children && (
-		<Box fontSize={2} height="100%">
+		<Flex
```

Comment:
This sounds like a major change, since in case someone was using `<span>`s or other inline elemenets, they will now show one per line, instead of next to each other.